### PR TITLE
Update README.md installation instructions

### LINKED ISSUE #242: Add composer installation docs to Readme
Please add `composer require erusev/parsedown:version` to the docs :)

It is a reliable way to install it

### PR DESCRIPTION
**Pull request by cebe**
updated installation instructions. fixes #242

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
:-1: This is not correct

**Comment by GrahamCampbell** [comment]
Should be:

```bash
$ composer require erusev/parsedown
```

**Comment by cebe** [comment]
If this works this means that it is not documented... 

```
$ composer help require
Usage:
 require [--dev] [--prefer-source] [--prefer-dist] [--no-progress] [--no-update] [--update-no-dev] [--update-with-dependencies] [--ignore-platform-reqs] [--sort-packages] [packages1] ... [packagesN]

Arguments:
 packages                    Required package with a version constraint, e.g. foo/bar:1.0.0 or foo/bar=1.0.0 or "foo/bar 1.0.0"

Options:
[...]

Help:
 The require command adds required packages to your composer.json and installs them
 
 If you do not want to install the new dependencies immediately you can call it with --no-update
 

```

**Comment by GrahamCampbell** [comment]
Send a pull request to document it then. It's standard practice to not include the version anymore. Composer will pick a good one for you. Examples:

If the latest version of the package you want is 0.4.6, then composer will pick "0.4.*".
If the latest version of the package you want is 1.7.2, then composer will pick "~1.7".

**Comment by cebe** [comment]
Sent a PR to composer to add the docs: https://github.com/composer/composer/pull/3641

**Comment by cebe** [comment]
@erusev if there is anything else to adjust here, please let me know.

File: README.md [hunk 1]
```diff
@@ -16,7 +16,12 @@ Better [Markdown](http://en.wikipedia.org/wiki/Markdown) parser for PHP.
 
 ### Installation
 
-Include `Parsedown.php` or install [the composer package](https://packagist.org/packages/erusev/parsedown).
+Download and include [Parsedown.php] or install the [composer package] by running the following command:
+
+    composer require erusev/parsedown
+
+[Parsedown.php]: https://raw.githubusercontent.com/erusev/parsedown/master/Parsedown.php "The Parsedown class file"
```
Comment (aidantwoods) [inline README.md]: The only thing I'm hesitating on here is encouraging downloading straight from `master`. Probably safer for the manual instructions is saying something like:
```markdown
1. Download "Source code" from the [latest release](https://github.com/erusev/parsedown/releases/latest)
2. Include `Parsedown.php`
```

Comment (cebe) [reply README.md]: fine with me, feel free to cherry-pick the commit and adjust it.

Comment (aidantwoods) [reply README.md]: Mind checking the box in the sidebar that let's me push changes to this PR? :)

**Pull request closed without merging**